src/Regions.py

```diff
@@ -61,7 +62,11 @@ def create_region(world: World, multiworld: MultiWorld, player: int, name: str,
         for location in locations:
             loc_id = world.location_name_to_id.get(location, 0)
             locationObj = ManualLocation(player, location, loc_id, ret)
+            eventLocationObj = ManualLocation(player,"["+location+"]",None,ret)
```

Comment:
Idk if we want to create a event loc and item for each location, might want to check for a new attribute in locations something like "create_event"

Reply:
actually now that I gened a game it does double the locations in the spoiler logs so its probably a good idea to only add those if required

Reply:
actually I found out you can hide locations from the spoiler list of location by using `eventLocationObj.show_in_spoiler = False`
it still show you getting the event when required in the log but it doesnt double the location names in the location list

Reply:
I wonder if it would make sense to not have the flag be on locations, but to build the list of events we need to create as we're validating the json's

Reply:
100% the events shouldn't be in the list, i can mark them as hidden, if i worked more on the worlds side of ap i'd have caught that myself